### PR TITLE
Homescreen

### PR DESCRIPTION
**Pull request by jfidiles**
As I've done some other things in the mean time, I've combined the first pull request -
 [header_and_footer](https://github.com/fathomlondon/fathomlondon.github.io/pull/1) with the necessary changes into a single. Hopefully it is alright!

On the next pull requests, I'll fix everything before adding new things.

### CODE REVIEW

File: _sass/main/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.Footer {
+  padding: $gap-large;
+  color: inherit;
```
Comment (artisologic) [inline _sass/main/_footer.scss]: That's unnecessary as it's the initial value.

File: _sass/main/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  font-weight: 200;
+  background-color: #F8F8F8;
+  display: flex;
+  flex-direction: row;
```
Comment (artisologic) [inline _sass/main/_footer.scss]: Same, that's the initial value.

File: _sass/main/_base.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+body {
+  margin: 0;
+  font-family: $base-font-family;
+  color: $color-text;
+  min-width: 320px;
+}
+
+h1 {
+  font-size: 2em;
```
Comment (artisologic) [inline _sass/main/_base.scss]: Did you generate a random scale here or is that based on the design?

File: _sass/main/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  @include media-query(small-device) {
+    border: none;
+    flex-direction: row;
+    padding: 0;
+  }
+}
+
+.Footer-twitterIcon,
```
Comment (artisologic) [inline _sass/main/_footer.scss]: If they are styled the same way, then a generic `.Footer-icon` might be clearer?

File: _sass/main/_home.scss [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+.Home-intro {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  min-height: 150px;
+  padding: $gap-large;
+
+  article span {
```
Comment (artisologic) [inline _sass/main/_home.scss]: This seems a bit too generic, I assume this is to highlight the words in the first statements at the top of the homepage? This is a perfect use-case for SuitCSS utility classes. You could for example create a utility class like so:

```css
.u-color-brand {
	color: $color-brand;
}
```

And you can use this in places like this where it's just a simple treatment.

File: _sass/main/_home.scss [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+    padding: $gap-large $gap-medium;
+  }
+
+  @include media-query(small-device) {
+    flex-direction: row;
+    justify-content: center;
+  }
+}
+
+.Home-aboutUsMainArticle {
```
Comment (artisologic) [inline _sass/main/_home.scss]: I think the names are getting quite verbose here, maybe you could try to qualify them a bit less and make them simpler?

File: _sass/main/_home.scss [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+  line-height: $line-spacing-medium;
+
+  @include media-query(small-device) {
+    max-width: 300px;
+    font-size: 0.875em;
+    margin: 0 $gap-xlarge 0 $gap-medium;
+  }
+}
+
+.Home-aboutUsIconsSection {
```
Comment (artisologic) [inline _sass/main/_home.scss]: For example, this could be `.Home-icons`

File: _sass/main/_home.scss [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+    margin: auto;
+    padding: $gap-xlarge 0;
+
+    div {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+
+    div img {
```
Comment (artisologic) [inline _sass/main/_home.scss]: I think here, either name things or use nesting completely.

File: _sass/main/_base.scss (hunk 3 above)
Comment (jfidiles) [inline _sass/main/_base.scss]: On the design I've got multiple sizes. But they are close to sizes found on the design.

**Pull request merged**